addons/skin.estuary/xml/Variables.xml

```diff
@@ -200,6 +200,7 @@
 	<variable name="SeekTimeLabelVar">
 		<value condition="!String.IsEmpty(Player.SeekNumeric)">[B]$INFO[Player.SeekNumeric(hh:mm:ss)][/B]</value>
 		<value condition="Player.Seeking">$INFO[Player.SeekTime]$INFO[Player.Duration, / ]</value>
+		<value condition="VideoPlayer.HasEpg">$INFO[PVR.Time]$INFO[PVR.Duration, / ]</value>
```

Comment:
ksooo has just asked me to inform you these infolabels are going to be removed. ;-p

Reply:
Yep.